django/contrib/admin/views/autocomplete.py

```diff
@@ -24,11 +26,29 @@ def get(self, request, *args, **kwargs):
             return JsonResponse({'error': '403 Forbidden'}, status=403)
 
         self.term = request.GET.get('term', '')
+        app_label = request.GET.get('app_label')
```

Comment:
This line and the following ones will throw a server error if the variables do not exist?